src/server/api-server.js

```diff
@@ -77,7 +78,7 @@ class APIServer extends EventEmitter {
   startListening() {
     return new Promise((resolve, reject) => {
       this._expressServer = this._expressApp.listen(
-        this._port, this._host, () => {
+        this._port, this._serveHost, () => {
```

Comment:
Can you just alter _host and then the other references will be using 0.0.0.0 as well.